go-controller/pkg/ovn/egressfirewall_dns.go

```diff
@@ -167,14 +167,17 @@ func (e *EgressDNS) Run(defaultInterval time.Duration) {
 	// initially the next DNS Query happens at the default interval
 	durationTillNextQuery := defaultInterval
 	go func() {
+		timer := time.NewTicker(durationTillNextQuery)
+		defer timer.Stop()
 		for {
+			timer.Reset(durationTillNextQuery)
```

Comment:
A bit of a nit here, but wouldn't it make more sense for this reset to be at the end of the for block, as it would be called unnecessarily on the first iteration and that's where actually the ticker period is re-calculated?